com.napster.napster-bigscreen-electron.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.napster.napster-bigscreen-electron
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+separate-locales: false
+command: start-napster-bigscreen-electron
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=/run/udev:ro
+
+modules:
+  - name: napster-bigscreen-electron
+    buildsystem: simple
+    build-commands:
+      - unzip -d napster-bigscreen-electron napster-bigscreen-electron_*.zip
+      - rm napster-bigscreen-electron_*.zip
+      - cp -r napster-bigscreen-electron /app
+      - install -Dm 644 com.napster.napster-bigscreen-electron.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm 644 com.napster.napster-bigscreen-electron.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 com.napster.napster-bigscreen-electron.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -Dm 755 start-napster-bigscreen-electron.sh /app/bin/start-napster-bigscreen-electron
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
```

Comment:
this doesn't prevent a aarch64 build.

Reply:
Removed

Reply:
the aarch64 build will be still going.

If you still don't want to build for aarch64 
https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
that would be disappointing though.

Reply:
I'll keep it without restricting to x86.

Reply:
@hfiguiere I'll take back my previous message. I just pushed a change that restricts to `x86_64` until we work internally with multi-arch support. We intend to have build instructions on the `yaml` in the next iteration of this software but I limited it just so we could move forward.